src/logic/tellMeJoke.js

```diff
@@ -0,0 +1,57 @@
+export const jokes = [
```

Comment:
Try to extract all of the so called static texts (jokes, answers, facts) to 1 file that would only contain them. This will be place where programmers are going only with the intention of updating them. Otherwise there is no need for them to see this if they want to manipulate just with the logic of tellingJokes/answering. It would make the files significantly smaller and easier to read and maby in future when you would get good enough generic level you would be able to merge them into 1 file with some methods (but for that we will see in the future, right now focus on the static texts extraction)

Reply:
I moved logic from (jokes, facts, answers) to textBox file and create new file cold textBox Data that's stor all of static texts.